topology/probes/netlink.go

```diff
@@ -79,6 +79,26 @@ type NetLinkProbe struct {
 	wg      sync.WaitGroup
 }
 
+// RouteTable describes a list of Routes
+type RoutingTable struct {
+	Id      int
+	IfIndex int
+	Src     net.IP
```

Comment:
you should add 'omitempty' where it is needed so that no empty field indexed and reported by the API

Reply:
done